Gopkg.toml

```diff
@@ -42,28 +42,42 @@ required = [
   name = "google.golang.org/grpc"
   version = "^1.8.2" # match istio/api
 
-[[constraint]]
+
+# When necessary, override transitive contraints that might be pinned
+# to older versions (e.g. ingress-nginx pinned to 1.8).
+
+[[override]]
   name = "k8s.io/client-go"
-  revision = "7c69e980210777a6292351ac6873de083526f08e"
+  branch = "release-6.0"
```

Comment:
Thank you !

Could we also add a date (when this was released) and a comment on typical release frequency 
if we know ? So people reading the file can get an idea how old the release is and if it needs refresh.

Reply:
https://github.com/istio/istio/pull/2416